docs/ardana/testing.md

```diff
@@ -181,17 +174,42 @@ To verify that the test environment is properly set up:
 openstack --os-cloud engcloud-cloud-ci stack list
 ```
 
-* run an ansible playbook, e.g.:
+### Manual Ardana deployment
+
+A bash script is provided to help deploying Ardana manually. The script sets up a virtual environment with ansible and all
+requirements and mimics the steps of the openstack-ardana job by calling the ansible playbooks in the appropriate
+sequence with the appropriate parameters, which are taken from `input.yml`.
+
+Before running the script you need to configure the parameters in the `input.yml` file to fit how you want ardana to be
+deployed. The options available are basically a subset of the inputs from the openstack-ardana jenkins job.
+
+**NOTE**: For building test packages from gerrit changes, the osc utility needs to be correctly installed and configured on
```

Comment:
Could you please move this note to the list under the `Test environment setup` section above ?

Reply:
done!